neo/rawio/blackrockrawio.py

```diff
@@ -597,6 +640,7 @@ def _get_timestamp_slice(self, timestamp, seg_index, t_start, t_stop):
         else:
             ts = np.math.ceil(t_start * self.__nev_basic_header['timestamp_resolution'])
             ind_start = np.searchsorted(timestamp, ts)
+            # SearchSorted does not apply. timestamp is not sorted!
```

Comment:
is this comment still up to date? I guess it's not, but it's not obvious to me.

Reply:
I don't remember who write this comment.
Is it me ?

Reply:
No, this stuff is introduced in this PR. I probably entered it in an early version of this PR.

Reply:
Yes, as far as I remember this came from you. But as it only refers to single segments, searchsorted should work anyway because spikes are sorted for each segment. So I just removed the comment.